hubspot/plugin.py

```diff
@@ -288,10 +635,205 @@ class HubSpotCompany:
+        "hs_user_ids_of_all_notification_unfollowers",
+        "hs_user_ids_of_all_owners",
+        "hs_was_imported",
+        "hubspot_owner_assigneddate",
+        "is_public",
+        "num_associated_contacts",
+        "num_associated_deals",
+        "num_conversion_events",
+        "recent_conversion_date",
+        "recent_conversion_event_name",
+        "recent_deal_amount",
+        "recent_deal_close_date",
+        "timezone",
+        "total_money_raised",
+        "total_revenue",
+        "name",
+        "owneremail",
+        "twitterhandle",
+        "ownername",
+        "phone",
+        "twitterbio",
+        "twitterfollowers",
+        "address",
+        "address2",
+        "facebook_company_page",
+        "city",
+        "linkedin_company_page",
+        "linkedinbio",
+        "state",
+        "googleplus_page",
+        "engagements_last_meeting_booked",
+        "engagements_last_meeting_booked_campaign",
+        "engagements_last_meeting_booked_medium",
+        "engagements_last_meeting_booked_source",
+        "hs_latest_meeting_activity",
+        "hs_sales_email_last_replied",
+        "hubspot_owner_id",
+        "notes_last_contacted",
+        "notes_last_updated",
+        "notes_next_activity_date",
+        "num_contacted_notes",
+        "num_notes",
+        "zip",
+        "country",
+        "hubspot_team_id",
+        "hs_all_owner_ids",
+        "website",
+        "domain",
+        "hs_all_team_ids",
+        "hs_all_accessible_team_ids",
+        "numberofemployees",
+        "industry",
+        "annualrevenue",
+        "lifecyclestage",
+        "hs_lead_status",
+        "hs_parent_company_id",
+        "type",
+        "description",
+        "hs_num_child_companies",
+        "hubspotscore",
+        "createdate",
+        "closedate",
+        "first_contact_createdate",
+        "days_to_close",
+        "web_technologies",
+    ]
+
+
+@dataclass
+class HubSpotCompanyCustomPropertyName:
+    name: str
+
+
+def _parse_company(data: dict) -> HubSpotCompany:
+    properties = data["properties"]
+
+    company_properties = HubSpotCompanyProperties(
+        name=properties["name"],
+        domain=properties["domain"],
+        hs_object_id=properties["hs_object_id"],
+        last_modified_date=datetime.fromisoformat(properties["hs_lastmodifieddate"]),
+        additional_properties={
+            key: value
+            for key, value in data["properties"].items()
+            if key not in ["name", "domain", "hs_object_id", "lastmodifieddate"]
+            and value is not None
+        },
+    )
+
+    return HubSpotCompany(
+        id=data["id"],
+        properties=company_properties,
+        createdAt=datetime.fromisoformat(data["createdAt"]),
+        updatedAt=datetime.fromisoformat(data["updatedAt"]),
+        archived=data["archived"],
+    )
+
+
 def list_companies(
     limit: int = 100,
     after: Optional[str] = None,
```

Comment:
i had an impression we used a different arg name for the cursor. 

can you check the rest of the codebase and make sure we're moving towards being consistent here?

Reply:
Updated all instances of `after` to `pagination_token`

Reply:
I initially chose 'after' because that's the name of the arg in the hubspot API / documentation -- my thought was that it's better to align with the documentation / names of the APIs being used, rather than our own.  But perhaps it doesn't matter.